content/rs/databases/active-active/manage-aa.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: Manage Active-Active databases 
+description: Manage Active-Active
+weight: 30
+alwaysopen: false
+categories: ["RS"]
+aliases: [
+    /rs/databases/active-active/edit-aa-database.md,
+    /rs/databases/active-active/edit-aa-database/,
+    /rs/databases/active-active/manage-aa.md,
+    /rs/databases/active-active/manage-aa/,
+]
+---
+
+You can configure and manage your Active-Active database from either the admin console or the command line.
+
+When you edit the database configurations of an Active-Active database with the admin console or the `rladmin` CLI, the changes will only apply to the Active-Active database instance you are currently editing.
+
+To change the global configuration of the Active-Active database, use the [`crdb-cli`]({{<relref "/rs/references/cli-utilities/crdb-cli">}}).
```

Comment:
I want us to recommend and push users to use the global, I'm assuming they don't know when they need global or local - and so they need global (as default) and sometimes local.
WDYT?